index.html

```diff
@@ -69,8 +73,124 @@
 
     <section class='informative'>
       <h2>Introduction</h2>
-      <p>This document is a stub.</p>
+      <p>The Browser Extensions Community Group's goal is to make browser extension code much more interoperable across browsers by specifying common extension interfaces and well-defined browser behavior. </p>
+      <p>This specification defines the interfaces and behavior which enables extensions to communicate with local client native processes outside of the browser environment.</p>
+    </section>
+
+    <section class='informative'>
+        <h2>Core APIs</h2>
+        <p>The goal for these APIs is to provide functionality that extension authors need for:</p>
+        <ul>
+            <li>message-passing between extensions and local client processes outside of the browser</li>
+            <li>message-passing to long-lasting connection-based ports</li>
+            <li>message-passing to connectionless processes</li>
+        </ul>
+        <section>
+            <h4 id="runtime">The <code>runtime</code> object</h4>
+            <section class="introductory">
+                <h4>Overview</h4>
+                <p>
+                    In addition to the base functionality of the <code>browser.runtime</code> object defined in the <a href="https://browserext.github.io/browserext#runtime">Browser Extensions specification</a>, these additional APIs provide functionaly for Native Messaging.
```

Comment:
Please use the respec syntax for making references to other specs: https://github.com/w3c/respec/wiki/User's-Guide#references, this will allow Respec to automatically generate the Normative and Informative References sections of the spec.